script/start

```diff
@@ -10,21 +10,27 @@ do
   fi
 done
 
+makeChecks() {
+  . "${DIR}"/lib/check
+}
+
 runChecker() {
   if [[ "$IS_DEBUG" = true ]]; then
-    bash "${DIR}"/start-checker --debug
+    bash "${DIR}"/start-checker $1 --debug
   else
-    bash "${DIR}"/start-checker
+    bash "${DIR}"/start-checker $1
   fi
 }
 
 runRuleManager() {
   if [[ "$IS_DEBUG" = true ]]; then
-    bash "${DIR}"/start-manager --debug
+    bash "${DIR}"/start-manager $1 --debug
   else
-    bash "${DIR}"/start-manager
+    bash "${DIR}"/start-manager $1
   fi
 }
 
-runChecker &
-runRuleManager
\ No newline at end of file
+# Make checks once now, rather than in each start script
+makeChecks
+runChecker "--checks-complete" &
```

Comment:
I considered calling this flag '--no-checks' but thought it might lead people to believe no checks are being carried out at all. They are, just in the top-level script. 

Then again, you can argue this flag doesn't make sense from within the context of the lower-level scripts. 🤷

Reply:
There are also other ways to pass this state than by using flags. Am open to suggestions!

NB. if any (unhandled) check in 'makeChecks' fails, it shouldn't get through to 'runChecker' and 'runManager', so passing the flags like this feels safe to me

Reply:
I think as long as API is clear and we're not going to set by accident somehow then 👍, this LGTM!